src/lib/tiptap-utils.ts

```
@@ -0,0 +1,537 @@
+
+  /**
+   * If enabled, it allows optional slashes after the protocol.
+   * @default false
+   * @example true
+   */
+  optionalSlashes?: boolean
+}
+
+type ProtocolConfig = Array<ProtocolOptions | string>
+
+const ATTR_WHITESPACE = /[\u0000-\u0020\u00A0\u1680\u180E\u2000-\u2029\u205F\u3000]/g
```

Biome (2.4.7)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)